zetaclient/orchestrator/orchestrator.go

```diff
@@ -518,3 +565,58 @@ func (oc *Orchestrator) ScheduleCctxBTC(
 		}
 	}
 }
+
+func (oc *Orchestrator) runObserverSignerSync(ctx context.Context) error {
```

Comment:
Might be good to give comment even if it's private method

Reply:
https://github.com/zeta-chain/node/pull/2497/commits/2f06e5639910af4fbfeb4261ff0faf4a86b53af1